src/executables/util_cs.c

```diff
@@ -1741,6 +1742,15 @@ tranlist (UTIL_FUNCTION_ARG * arg)
       db_shutdown ();
       goto error_exit;
     }
+#else
+  AU_DISABLE_PASSWORDS ();
+  db_set_client_type (DB_CLIENT_TYPE_ADMIN_UTILITY);
+  if (error = db_login ("DBA", NULL) || (error = db_restart (arg->command_name, TRUE, database_name)))
```

Comment:
I cannot clearly understand. The value of the variable '**error**' will always be 0 or 1. And I think you don't use value of error anymore. I think it would be better to modify it as follows:
```
either
if (db_login ("DBA", NULL) || db_restart (arg->command_name, TRUE, database_name))
or
if ((error = db_login ("DBA", NULL)) || (error = db_restart (arg->command_name, TRUE, database_name)))
```

Reply:
Thank you for review. I will remove unneeded variable.